src/hotspot/share/jfr/jfr.cpp

```diff
@@ -112,9 +112,15 @@ bool Jfr::on_start_flight_recording_option(const JavaVMOption** option, char* de
   return JfrOptionSet::parse_start_flight_recording_option(option, delimiter);
 }
 
-#ifdef JFR_HAVE_INTRINSICS
 JRT_LEAF(void, Jfr::trace_id_load_barrier(Klass * klass))
```

Comment:
please declare as const, as in Jfr::trace_id_load_barrier(const Klass* klass);"

Reply:
fixed